TestFramework/Tests.Integration/Tests/UserIntegrationTests.cs

```diff
@@ -345,7 +346,8 @@ public void TestMIC_LoginWithMIC_NormalFlow()
 		}
 
 		[Test]
-		public void TestMIC_LoginWithMIC_NormalFlow_ClientID()
+        [Obsolete("This test has been deprecated.")]
+        public void TestMIC_LoginWithMIC_NormalFlow_ClientID()
```

Comment:
We are not deprecating the normal MIC flow (Authorization Grant Flow), why is this test method being deprecated?